crates/swc_macros_common/Cargo.toml

```diff
@@ -15,4 +15,4 @@ quote = "1"
 
 [dependencies.syn]
 features = ["derive", "visit", "parsing", "full", "printing", "extra-traits"]
-version = "=1.0.65"
+version = "=1.0.84"
```

Comment:
wasmer-derive have conflict semver requirements to this which blocks compilation.